app/helpers/sessions_helper.rb

```diff
@@ -13,6 +14,20 @@ def logged_in?
 
   def log_out
     session.delete :user_id
+    cookies.delete :user_id
```

Comment:
log in bằng cookies hả em sao xóa thế

Reply:
login bằng session anh...còn cái cookies này em lưu để lấy định danh kết nói với websocket